### PR TITLE
Add stylings to All Components

### PR DESCRIPTION
**Pull request by GhausAliKhan**
In this Branch I did:  

- Add styles to Match the [Zeplin's Design](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/images/bookstore.png)

### CODE REVIEW

File: src/styles/navbar.css [hunk 1]
```diff
@@ -19,3 +41,26 @@ header {
...
+  font-family: "Roboto", sans-serif;
+  font-size: 1.1rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  text-decoration: none;
+  color: #121212;
+}
```
Comment (Alusp) [inline src/styles/navbar.css:58]: - [x] You have done really well so far, :clap:, however, according to the design requirement, `Home` and `Categories`  texts styles are a bit diferent, The `Categories` text has an opacity of 0.5 while `Home` does not have any opacity. Matching your design to the given design requirements enhances the chances of creating a successful and well-rounded design solution that meets the needs of users, complies with regulations, and performs effectively and efficiently. :slightly_smiling_face: 

Below is an illustration of yours and the given Design, please pay attention to the red markings on your design: :arrow_down: 

<table>
    <tr>
      <td>Your Design</td>
      <td>Given Design</td>
    </tr>
    <tr>
      <td>

StudentNav

</td>
      <td>

epNav


</td>
    </tr>
  </table>

**Review by Alusp — CHANGES_REQUESTED** [review]
Hi @GhausAliKhan, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
close

 ## Highlights

- [x] Linters set-up ✔️
- [x] Commit messages descrptive ✔️
- [x] Professional Github Flow used ✔️
- [x] Professional PR Summary Flow used ✔️
- [x] UI Closer to the given design :heavy_check_mark:  

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alusp in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/navbar.css (hunk 1 above)
Comment (GhausAliKhan) [reply src/styles/navbar.css:58]: Thank you for your review, I've added the changes 👍🏻

**Review by V-Blaze — APPROVED** [review]
Hi @GhausAliKhan  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- UI looks like the given design💯
-  App works as expected👍
-  NO inline styled component 🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by GhausAliKhan** [comment]
Appreciate that!

File: src/styles/navbar.css (hunk 1 above)
Comment (Alusp) [reply src/styles/navbar.css:58]: You are most welcome 🙂

**Pull request merged**